frontend/src/components/CategoryFilter/CategoryFilter.tsx

```diff
@@ -38,20 +39,20 @@ export class CategoryFilter extends React.Component<
           </div>
           <div className="filter__input">
             <label htmlFor="date">Datum</label>
-            <div className="input__radio"><input type="radio" id="date" /> <span>Heute</span></div>
-            <div className="input__radio"><input type="radio" id="date" /> <span>Morgen</span></div>
-            <div className="input__radio"><input type="radio" id="date" /> <span>Nächste Woche</span></div>
+            <div className="input__radio"><input type="radio" name="date" id="today" /> <label htmlFor="today">Heute</label></div>
+            <div className="input__radio"><input type="radio" name="date" id="tomorrow" /> <label htmlFor="tomorrow">Morgen</label></div>
+            <div className="input__radio"><input type="radio" name="date" id="week" /> <label htmlFor="week">Nächste Woche</label></div>
           </div>
         </div>
-        <div className="col col-lg-2">
+        <div className="col col-lg-3">
           <div className="filter__input">
             <label htmlFor="activity">Aktivitäten</label>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Gottesdienst</span></div>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Jugend</span></div>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Musik</span></div>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Gebete</span></div>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Senioren</span></div>
-            <div className="input__checkbox"><input type="checkbox" id="activity" /> <span>Kurse</span></div>
+            <div className="input__checkbox"><input type="checkbox" id="activity-1" value="Gottesdienst" /> <label htmlFor="activity-1">Gottesdienst</label></div>
```

Comment:
In React erzeugt man hier üblicherweise eine Komponente <Checkbox>, die dann eine Checkbo darstellt. Das hat auch den Vorteil, dass man den state der Checkbox dann gut gekapselt handlen kann.
```suggestion
            <Checkbox onChange={handleServiceFlas}>Gottesdienst</Checkbox>
```